frontend/components/brand/dashboard-layout/dashboard-navbar.tsx

```
@@ -0,0 +1,33 @@
+import { usePathname } from "next/navigation"
+import { cn } from "@/lib/utils"
+import { useAuth } from "@/provider/useAuth"
+
+export default function DashboardNavbar() {
+  const pathname = usePathname()
+  const { userRole } = useAuth()
+  return (
+    <div className="bg-gray-50 py-4 border-b">
+      <nav className="flex items-center gap-4 container mx-auto p-2">
+        {dashboardMenu
+          .filter((item) => item.role.includes(userRole!))
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)